product/infrastructure/fake/searchservice.go

```diff
@@ -167,6 +167,24 @@ func (s *SearchService) findCurrentPage(filters []searchDomain.Filter) int {
 	return currentPage
 }
 
+func (s *SearchService) findSorts(filters []searchDomain.Filter) []searchDomain.SortOption {
+	result := make([]searchDomain.SortOption, 0)
+	// first check if the searchDomain.SortFilter is given
+	for _, v := range filters {
+		switch filter := v.(type) {
+		case *searchDomain.SortFilter:
+			result = append(result, searchDomain.SortOption{
+				Label:        filter.Field(),
+				Field:        filter.Field(),
+				SelectedAsc:  !filter.Descending(),
+				SelectedDesc: filter.Descending(),
+			})
+		default:
+		}
```

Comment:
switches with only one case should be an if statement:
```suggestion
		if filter,ok := v.(*searchDomain.SortFilter); ok {
			result = append(result, searchDomain.SortOption{
				Label:        filter.Field(),
				Field:        filter.Field(),
				SelectedAsc:  !filter.Descending(),
				SelectedDesc: filter.Descending(),
			})
		}
```